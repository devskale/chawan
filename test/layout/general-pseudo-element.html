<!DOCTYPE html>
<style>
::before { content: "test"; color: red }
</style>
<div>test</div>
