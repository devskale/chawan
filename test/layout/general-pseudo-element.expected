[38;2;255;40;40mtest[39m
[38;2;255;40;40mtest[39m
[38;2;255;40;40mtest[39mtest
